CHANGELOG.md

```diff
@@ -21,6 +21,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Widgets can now have an ALLOW_CHILDREN (bool) classvar to disallow adding children to a widget https://github.com/Textualize/textual/pull/3758
 - Added the ability to set the `label` property of a `Checkbox` https://github.com/Textualize/textual/pull/3765
 - Added the ability to set the `label` property of a `RadioButton` https://github.com/Textualize/textual/pull/3765
+- Added support for Ctrl+Fn and Ctrl+Shift+Fn keys in urxvt https://github.com/Textualize/textual/pull/3737
+- Added support for various modified edit and navigation keys in urxvt https://github.com/Textualize/textual/pull/3739
```

Comment:
Which ones? Are these also the Ctrl + Fn keys added below?

Reply:
https://github.com/Textualize/textual/pull/3737/commits/a5650f73c4d5d9465cecb9b7b79aa3c9e7517619